x-pack/legacy/plugins/maps/public/shared/layers/sources/ems_tms_source/ems_tms_source.js

```diff
@@ -117,4 +120,16 @@ export class EMSTMSSource extends AbstractTMSSource {
     const emsTmsMeta = await this._getEmsTmsMeta();
     return emsTmsMeta.url;
   }
+
+  _getEmsTileLayerId() {
+    if (this._descriptor.id !== EMS_TILE_AUTO_ID) {
```

Comment:
should we make the change to the descriptor, this implementaiton is going to toggle on the mode-property instead